Sources/GRPCCore/GRPCServer.swift

```diff
@@ -206,10 +206,10 @@ public struct GRPCServer: Sendable {
   /// Signal to the server that it should stop listening for new requests.
   ///
   /// By calling this function you indicate to clients that they mustn't start new requests
-  /// against this server. Once the server has processed all requests the ``run()`` method returns.
+  /// against this server. Once the server has processed all requests the ``serve()`` method returns.
   ///
   /// Calling this on a server which is already stopping or has stopped has no effect.
-  public func stopListening() {
+  public func beginGracefulShutdown() {
```

Comment:
I would suggest using `triggerGracefulShutdown` to align with https://github.com/swift-server/swift-service-lifecycle/blob/24c800fb494fbee6e42bc156dc94232dc08971af/Sources/ServiceLifecycle/ServiceGroup.swift#L167.